solutions/usaco-832.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: usaco-832
+source: USACO Bronze 2018 Open
+title: Milking Order
+author: Ananth Kashyap
+---
+
+## Solution 1
+
+[Official Analysis](http://www.usaco.org/current/data/sol_milkorder_bronze_open18.html)
+
+## Solution 2
+
+In this problem, there are three cases that we have to consider: 
+- The first of these cases is that cow one is fixed. In this case, the solution is simple, as we just output its fixed position. 
+- The second case we must consider is if cow one is in the given hierarchy of cows. In this case, we want to place every cow in the hierarchy as early as possible. 
+- The last case is when cow one is neither fixed nor in the hierarchy. In this case, we would want to place every cow in the hierarchy as late as possible, allowing for cow one to get the earliest position possible.
+<LanguageSection>
+<PySection>
+
+## Python Implementation
+
+```py
+file_in = open('milkorder.in')
+data = file_in.read().strip().split('\n')
+n, m, k = map(int, data[0].split(' '))
+hierarchy = list(map(int, data[1].split(' ')))
+fixed = [list(map(int, data[i + 2].split(' '))) for i in range(k)]
+
+cows = [0 for i in range(n)]
+
+for i in range(k):
+  	cows[fixed[i][1] - 1] = fixed[i][0]
+
+if 1 in cows:
```

Comment:
can you add comments here? like `# case 1/2/3`

Reply:
Will do